src/api.ts

```diff
@@ -104,10 +101,10 @@ function getExploreResult(
   results: (ExploreBundleResult | ExploreErrorResult)[],
   options: ExploreOptions
 ): ExploreResult {
-  const [bundles, errors] = partition(results, result => 'files' in result) as [
-    ExploreBundleResult[],
-    ExploreErrorResult[]
-  ];
+  const [bundles, errors] = partition(
+    results,
+    (result): result is ExploreBundleResult => 'files' in result
```

Comment:
I was impressed that the lodash type declarations for partition knew about type guards… then I saw that you added that :)